### PR TITLE
User support

### PR DESCRIPTION
**Pull request by NikitaDocent**

### CODE REVIEW

**Comment by ghost** [comment]
Why 2 persons were working on this branch ?
Should I divide your mark by 2 ? ;)

**Comment by NikitaDocent** [comment]
Not , we work with repo on one computer , and when choosing accounts in github for work , didn’t choose name settings , tomorrow , I display you how it work in university.

File: App.Common/App.Common.csproj [hunk 1]
```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Castle.Core" Version="4.4.0" />
```
Comment (NikitaDocent) [inline App.Common/App.Common.csproj]: delete package

File: App.Common/Repositories/IUserSupportMessagesRepository.cs [hunk 2]
```diff
@@ -1,6 +1,11 @@
-﻿namespace App.Repositories
+﻿using System.Collections.Generic;
+
+namespace App.Repositories
 {
     public interface IUserSupportMessagesRepository
     {
+
+        IEnumerable<string> GetValues();
```
Comment (NikitaDocent) [inline App.Common/Repositories/IUserSupportMessagesRepository.cs]: Name is not correct

File: App.TestModule/App.Example.csproj [hunk 3]
```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Castle.Core" Version="4.4.0" />
```
Comment (NikitaDocent) [inline App.TestModule/App.Example.csproj]: delete

File: App.UserSupport/Controllers/UserSupportController.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+        [HttpGet("{id}/last10Message")]
+        public ActionResult<IEnumerable<string>> GetHandling10LastMessages(int id)
+        {
+            var serviceCallResult = _userSupportManager.GetHandling10LastMessages(id).ToList();
+            return serviceCallResult;
+        }
+
+        [HttpPost]
+        public ActionResult<IEnumerable<string>> SetHandlingStatusCompletedAndReturnListActiveHandlings(int id)
```
Comment (NikitaDocent) [inline App.UserSupport/Controllers/UserSupportController.cs]: too long method  namme

File: App.UserSupport/Models/Handling.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        {
+            Id = id;
+            status = false;
+            clientHandling = client;
+            WriteMessage(clientHandling.Name, firstmessage);
+        }
+
+        public void SetHandlingStatusCompleted() => status = true;
+
+        public void WriteMessage(string mess)
```
Comment (NikitaDocent) [inline App.UserSupport/Models/Handling.cs:22]: move BL to manager

File: App.UserSupport/Repositories/InMemoryUserSupportRepository.cs [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+using App.UserSupport.Models;
+
+namespace App.UserSupport.Repositories
+{
+    public interface IUsersRepository
+    {
+        IEnumerable<string> GetStringListActiveValues();
+        Handling Get(int id);
+    }
+    public class InMemoryUsersRepository : IUsersRepository, ITransientDependency
```
Comment (NikitaDocent) [inline App.UserSupport/Repositories/InMemoryUserSupportRepository.cs]: change name

File: App.UserSupport/Services/AnotherService.cs [hunk 7]
```diff
@@ -0,0 +1,15 @@
+﻿namespace App.UserSupport
```
Comment (NikitaDocent) [inline App.UserSupport/Services/AnotherService.cs]: delete

File: App.UserSupport/UserSupportModule.cs [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+using Castle.MicroKernel.Registration;
+using Castle.Windsor;
+
+namespace App.UserSupport
+{
+    public class UserSupportModule : IModule
+    {
+        public void Initialize(IWindsorContainer container)
+        {
+            container.Register(Component.For<IAnotherService>().ImplementedBy<AnotherService>().LifestyleTransient());
```
Comment (NikitaDocent) [inline App.UserSupport/UserSupportModule.cs]: delete

File: App.Web/App.Web.csproj [hunk 9]
```diff
@@ -21,7 +21,6 @@
     <PackageReference Include="Castle.Windsor" Version="5.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
```
Comment (NikitaDocent) [inline App.Web/App.Web.csproj]: reestore

File: App.Web/Modules.cs [hunk 10]
```diff
@@ -2,11 +2,10 @@
 
 namespace App.Web
 {
-    /// <summary>
```
Comment (NikitaDocent) [inline App.Web/Modules.cs:5]: return

**Review by ghost — APPROVED** [review]

**Pull request merged**